packages/mobile/ios/Podfile.lock

```diff
@@ -814,7 +814,7 @@ SPEC CHECKSUMS:
   Permission-Notifications: 2662b6885535a98892b8e48da04b74fc898d70f8
   PromisesObjC: 68159ce6952d93e17b2dfe273b8c40907db5ba58
   Protobuf: 235750e4696ff59fb07d949a9dbbc92b3c0700fe
-  RCT-Folly: 803a9cfd78114b2ec0f140cfa6fa2a6bafb2d685
+  RCT-Folly: a21c126816d8025b547704b777a2ba552f3d9fa9
```

Comment:
Been seeing these change a lot... I wonder if you were to install fresh what would happen?

Reply:
Let me try removing pods

Reply:
Yeah I still have the same change, I'll leave it in for now